notebooks/ch14/bias_modeling.md

```diff
@@ -215,14 +254,20 @@ plt.title('Irreducible error');
+
+As we have seen, however, achieving a low training error does not necessarily mean that our model will have a low test error as well. It is easy to obtain a model with extremely low bias and therefore low training error by fitting a curve that passes through every training observation. However, this model will have high variance, which generally leads to high test error. Conversely, you can obtain a model with very low variance by fitting a horizontal line to the data, but this leads to high bias. Fundamentally, this occurs because training error reflects the bias of our model but not the variance; the test error reflects both. In order to minimize test error, our model needs to simultaneously achieve low bias and low variance. To account for this, we need a way to simulate test error without using the test set. This is generally done using cross validation.
+
+Cross-validation provides a more accurate method of estimating our model error using a single observed dataset by separating data used for training from the data used for model selection and final accuracy. Intuitively, the validation or test error checks the model's performance on a previously unseen dataset and thus allows us to estimate both bias and variance. Cross validation also allows us to incorporate the fact that the noise in the training set only affects the noise term in the risk, whereas the noise in the training set only affects bias and model variance. To choose the final model to use, we select the one that has the lowest validation error.
+
+The simplest form of cross-validation is the **train-validation split**. However, this method is prone to high variance because the evaluation of the error may depend heavily on which points end up in the training and validation sets. To tackle this problem, we can run the train-validation split multiple times on the same dataset using  **k-fold cross-validation**. The biggest advantage of this method is that every data point is used for validation exactly once and for training *k-1* times. Typically, a *k* between 5 to 10 is used, but *k* remains an unfixed parameter. When *k* is small, the error estimate has a lower variance (many validation points) but has a higher bias (fewer training points). Vice versa, with large *k* the error estimate has lower bias but has higher variance. K-fold cross-validation outputs a prediction error that more accurately reflects the bias and variance of a model. In particular, the model bias is the validation error. If this value is low, our model predicts values with low error on average. The model variance can be estimated by taking the standard error of the errors in each of the *k* trials. If this value is high, our model's performance varies a lot with the dataset used for training. 
```

Comment:
Let's be really careful of using "variance" in this section since right now, the reader associates "variance"  with "model variance". In your second sentence, you're referring to the variance of the validation error which is not immediately obvious. I don't think we should talk about validation error bias / variance at all in this section because of this potential confusion. Although certainly an important topic, I think it fits better in the section on CV since choosing k is relevant for performing CV and not as relevant for managing the bias-variance tradeoff. Also, I don't think we need to re-introduce train-validation split / k-fold CV in great detail since these two topics have an entire section dedicated to explaining them in 13.3.

In practice, I haven't seen the subsample validation errors of k-fold CV used to manage the bias-variance tradeoff. Although the variance of these errors gives an estimate of the model variance, I find it easier to just plot validation error vs. model complexity since that's the number I really care about. Just knowing the model variance isn't helpful unless I know the bias too; although I can't decompose the bias/variance from validation error alone, I don't really need the individual bias/variance terms if I have the validation error.

With that being said, what do you think about this paragraph? Is there anything here you think is still important to keep in this section?

Reply:
I'll just get rid of this paragraph, and we can add the stuff about k in k-fold cross and the validation error/bias and variance in 13.3?